src/ga4gh/vrs/extras/translator.py

```diff
@@ -28,7 +28,7 @@
 class ValidationError(Exception):
     """Class for validation errors during translation"""
 
-    pass
+    pass  # pylint: disable=unnecessary-pass
```

Comment:
you actually don't need this line at all (idk if Pylint will accept that though?)

Reply:
Ya, I just do it out of habit. I'll remove and try to get better at not defaulting to pass

Reply:
honestly I think also it looks better with "pass" but having to annotate with a pylint ignore is just a nuisance/makes it uglier again. I dunno.

Reply:
Haha I agree about it looking nicer with pass. What way do you want to go? 😂